### PR TITLE
Word difficulty

### PR DESCRIPTION
**Pull request by Stephen00**
I've changed the source code such that each word is now based on difficulty with the specified ranges that you listed.

Here is a screenshot of the program working:

image

### CODE REVIEW

**Comment by noturshek** [comment]
Oh... I thought you closed the previous PR as you no more wanted to work on this program 😅 . Instead of closing the prev PR and creating a new PR you could have just rebase and squashed the commits. BTW have you checked this program for the i level?

**Comment by Stephen00** [comment]
Everything is fixed and should work correctly @itsabhianant and I'm glad this code helps you! 

also when merging you can squash the commits into a single commit like so:

image

Kind Regards,
Stephen

**Comment by noturshek** [comment]
Hi @Stephen00 ,

I think you need to update your branch once as some changes has been made to the master branch since you last forked the program.

**Comment by noturshek** [comment]
**The ```TODO``` of the master branch:**

Screenshot (7)

**The ```TODO``` of your branch:**
Screenshot (8)


Also make sure that you have deleted the TODO that you have done.

File: hangman.py [hunk 1]
```diff
@@ -1,34 +1,47 @@
...
 
-
-def get_valid_word(words):
+def get_valid_word(words, lives):
     """Gets word from words.py wihtout '-' or ' '."""
+    word = ""
 
-    word = random.choice(words)
+    if lives == 12:
+        word_dict = [word for word in words if len(word) >= 4 and len(word) <=6]
```
Comment (noturshek) [inline hangman.py]: As you can see here the m level still gives words of 6 letters. I don't want that. A minor change needs to be done.

Screenshot (11)

The following code will make it happen:
Screenshot (12)


Just make your ```get_valid_word``` look something like this and then the code will be perfectly ready to go.

**Review by noturshek — CHANGES_REQUESTED** [review]
I might sound like a boomer for making you constantly change things but we are so close to making it I have asked you to make some changes. Please look at them. Also please update the TODO. I really appreciate your approach of contributing to this program and once you make the suggested changes in ```get_valid_words``` and update the ```TODO``` your code is good to get merged.

File: hangman.py (hunk 1 above)
Comment (Stephen00) [reply hangman.py]: I've updated it now

**Comment by noturshek** [comment]
Thanks @Stephen00

**Pull request merged**